demo/src/app/components/shared/examples-page/demo.component.ts

```diff
@@ -20,8 +27,8 @@ export class NgbdWidgetDemoComponent {
   get markupSnippet() { return Snippet({lang: 'html', code: this.markup}); }
   get codeSnippet() { return Snippet({lang: 'typescript', code: this.code}); }
 
-  getFileSnippet({name, source}) {
-    return Snippet({code: source, lang: name.split('.').pop()});
+  getFileSnippet({name, source}: {name: string, source: string}) {
```

Comment:
Can't we use `ISnippet` interface here as a return type ?

Reply:
Will check

Reply:
We can indeed!